.github/workflows/bump-versions.yml

```diff
@@ -28,12 +39,24 @@ jobs:
         working-directory: ./src
         run: go run ./cmd/bump-versions
 
+      - name: Get branch
+        id: get-branch
+        run: |
+          if [ "${{ inputs.environment }}" = "Production" ]; then
+            export BRANCH="${{ github.ref }}"
+          elif [ "${{ inputs.environment }}" = "Development" ]; then
+            TIMESTAMP=$(date +%s)
+            export BRANCH="${{ github.ref }}-bump-versions-$TIMESTAMP"
+          fi
+          echo "BRANCH=$BRANCH" >> $GITHUB_ENV
+
       - uses: stefanzweifel/git-auto-commit-action@v5
         with:
           file_pattern: 'modules/**/* providers/**/*'
           commit_user_name: Auto Version Bumper
+          branch: ${{ env.BRANCH }}
+          create_branch: true
```

Comment:
Does this work correctly (not create a branch) when the branch already exists? Making sure this won't break the bump versions for prod

Reply:
Yep, I already tested it (here's an [example](https://github.com/Evi1Pumpkin/registry/actions/runs/7177197309/job/19543324980)).